src/internal/task/futex_linux.c

```diff
@@ -0,0 +1,28 @@
+//go:build none
+
+// This file is manually included, to avoid CGo which would cause a circular
+// import.
+
+#include <stdint.h>
+#include <sys/syscall.h>
+#include <time.h>
+#include <unistd.h>
+
+#define FUTEX_WAIT    0
+#define FUTEX_WAKE    1
+#define FUTEX_PRIVATE 128
```

Comment:
nit: also afaik the flag itself is called [`FUTEX_PRIVATE_FLAG`](https://elixir.bootlin.com/linux/v6.11/source/include/uapi/linux/futex.h#L26), just to be consistent with [`FUTEX_WAIT`](https://elixir.bootlin.com/linux/v6.11/source/include/uapi/linux/futex.h#L11) and `FUTEX_WAKE`

Reply:
Thanks! Fixed locally, will be included the next time I push the branch.